tests/hikari/models/test_users.py

```diff
@@ -33,6 +38,123 @@ def test_PremiumType_str_operator():
     assert str(type) == "NITRO_CLASSIC"
 
 
-def test_PartialUser_str_operator():
-    mock_user = mock.Mock(users.PartialUserImpl, username="thomm.o", discriminator="8637")
-    assert users.PartialUserImpl.__str__(mock_user) == "thomm.o#8637"
+class TestUser:
+    @pytest.fixture
+    def obj(self):
+        class StubUser(users.User):
```

Comment:
might want to look at the stub_class helper as well

Reply:
I initially discarded stub_class because it instantiates the `user.User` class, which it cant because it has abstractmethods